### DISCUSSION TITLE
PROBLEMS IN TIMELINE: When you increase the video playback speed (L key) in the Timeline, the sound disappears.

### DISCUSSION THREAD

**Discussion by talebegithub** [olive-editor/olive#2303] [General]
When increasing the playback speed of the video using the L key in the Timeline, the video speeds up, but the audio cuts off. This issue is not present in version 1. Additionally, when I pause and resume a playing video in the Timeline using the space key, it is not smooth; there is a significant delay. Olive v1 and Davinci Resolve perform much better in this regard. Despite several notable improvements in Olive v2, fundamental functions are worse compared to v1. Please prioritize fixing these basic functions.

**Comment by thomaspeterson345** [comment]
Here are some potential steps you could take to address these issues:

Check for Updates: Make sure you're using the latest version of Olive. Sometimes, these issues can be resolved with software updates that address bugs and improve performance.

Adjust Playback Settings: Explore the playback settings in Olive to see if there are any options related to audio synchronization or buffering that could be adjusted to improve performance.

Report Bugs: If you haven't already, consider reporting these issues to the developers of Olive. Providing detailed information about your system specs, the specific steps to reproduce the issue, and any error messages you encounter can help them diagnose and fix the problem more effectively.

Explore Alternative Workflows: In the meantime, you might consider using alternative methods for tasks that are affected by these issues. For example, you could try using external audio editing software to adjust the timing of audio clips separately from video playback in Olive.

Temporary Workarounds: Depending on the nature of the issues you're experiencing, there may be temporary workarounds that can help mitigate the impact until a permanent solution is implemented. For example, you could try pre-rendering portions of your timeline to reduce the strain on the playback engine.

By following these steps and potentially reaching out to the Olive community or developers for support, you can hopefully address these issues and continue using Olive effectively for your video editing needs.
